src/vlt-json/test/index.ts

```diff
@@ -337,3 +337,53 @@ t.test('do not walk up past a .git folder', async t => {
   unload()
   t.equal(load('root', assertBool), false)
 })
+
+t.test(
+  'create directories when saving to non-existent path',
+  async t => {
+    const dir = t.testdir({
+      a: {
+        b: {
+          'package.json': JSON.stringify({}),
+          c: {},
+        },
+      },
+    })
+    t.chdir(resolve(dir, 'a/b/c'))
+
+    const { load, save, find } = await t.mockImport<
+      typeof import('../src/index.ts')
+    >('../src/index.ts', {
+      'node:os': t.createMock(await import('node:os'), {
+        homedir: () => t.testdirName,
+      }),
+    })
+
+    // The find function should return a path that doesn't exist yet
+    const configPath = find()
```

Comment:
Find should likely point to the "user" profile

```suggestion
    const configPath = find('user')
```